ts/src/time-profiler.ts

```diff
@@ -78,13 +78,19 @@ export function start(
   // undocumented API.
   // See https://github.com/nodejs/node/issues/19009#issuecomment-403161559.
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  (process as any)._startProfilerIdleNotifier();
+  if (typeof (process as any)._startProfilerIdleNotifier === 'function') {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (process as any)._startProfilerIdleNotifier();
```

Comment:
Is this still needed if `(idle)` is no longer reported?

Reply:
They're not related.

Reply:
This is needed to identify `(program)` time correctly. If we want to _also_ remove `(program)` time though, it could be removed.